smoketest/systemlimits_test.go

```diff
@@ -159,31 +204,111 @@ func TestSystemLimits(t *testing.T) {
 				7. set limit to -1
 				8. create (should work)
 			*/
+			t.Parallel()
+			h := harness.NewHarness(t, sql, "limit-configuration")
+			defer h.Close()
+
+			doQL := func(t *testing.T, query string) (string, string) {
```

Comment:
I think we can move these helpers out of the t.Run scope and flatten things a bit, for this one we'll just need to add `h` as an argument.